crud/common/utils.lua

```diff
@@ -199,11 +199,47 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
-function utils.convert_operations(user_operations, space_format)
-    if utils.tarantool_supports_fieldpaths() then
-        return user_operations
+local function id_not_in_table(operations, id)
+    for _, operation in ipairs(operations) do
+        if operation[2] == id then
+            return false
+        end
+    end
+
+    return true
+end
+
+local function add_nullable_fields_rec(operations, space_format, tuple, id)
+    if id < 2 or tuple[id - 1] ~= box.NULL then
+        return operations
+    end
+
+    if space_format[id - 1].is_nullable and id_not_in_table(operations, id - 1) then
+        table.insert(operations, {'=', id - 1, box.NULL})
```

Comment:
What if I have tuple `{1}` and tried to do "tuple:update({{'=', 4, 'value'}})", adding of only single field won't work - you need to add two nulls.

Reply:
More complex case - `update({{'=', 4, value1}, {'=', "some_field", value2}})` and "some_field" is 6th field.